app/scss/_item-header.scss

```diff
@@ -0,0 +1,56 @@
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// Item header
+//
+// Displays the properties of the item at the
+// top of the dropdown area.
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+$item-header-spacing: 5px;
+
+.item-header {
+  background-color: $white;
+  color: $black;
+  letter-spacing: 1px;
+  overflow: hidden;
+  padding: $item-header-spacing 28px $item-header-spacing $item-header-spacing;
+  position: relative;
+  text-overflow: ellipsis;
+  white-space: nowrap !important;
+
+  a {
+    color: $black;
+  }
+}
+
+.item-header__aside {
```

Comment:
I can switch away from BEM. But I have noticed that there's not really a method for naming classes (it's pretty random). Thought I'd introduce this to the team to see how they like it.

For now I can adjust to some sort of child-selector CSS names.